### ISSUE TITLE
Decreased B cell count

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6082]
**Preferred term label:**
Decreased B cell count

**Synonyms**
B cell deficiency

**Definition (free text, please give PubMed ID)**
A lack of B cells that has been associated with selected clinical conditions, including immune cytopenias, splenomegaly, granulomatous disease and lymphadenopathy. 
PMID:  20477641

**Parent term (use hpo.jax.org/app)**
Abnormal B cell count 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:  614700

**Gene association**
LRBA (OMIM:606453 )

**Comment by LCCarmody** [comment]
Hi, Thanks for the notice.  Maybe we could work on the structure or synonyms, but the term 'B lymphocytopenia' HP_0010976 with the definition 'Abnormal decrease from the normal count of B cells' exists.

Syn: 'Low B cell count', 'reduction of B cell number'.  Could we add synonyms to this term?

**Comment by johannaest** [comment]
Hello,

Yes it would be great to add "Decreased B cell count" and "B cell deficiency" as synonyms to this term. 
It would also be good to add this term to the parent term "Abnormal B cell count" since there is already "increased B cell count" under that category.

**Comment by pnrobinson** [comment]
Thanks, Revising https://hpo.jax.org/app/browse/term/HP:0010976

**Comment by pnrobinson** [comment]
add synoynms and parentage
